pkg/controller/workload/job/job_controller.go

```diff
@@ -207,10 +219,25 @@ func (r *JobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 		}
 		log.V(3).Info("Job is suspended and workload not yet admitted by a clusterQueue, nothing to do")
 		return ctrl.Result{}, nil
+	} else {
+		log.Info("Job is running", "enabled", r.waitForPodsReady, "ready", jobPodsReady(&job))
```

Comment:
only log when the value changed? Also, at least V(3)

Reply:
Dropped it entirely